devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java

```diff
@@ -136,10 +136,18 @@ public Property<Boolean> getCacheLargeArtifacts() {
     public void setCacheLargeArtifacts(boolean cacheLargeArtifacts) {
         this.cacheLargeArtifacts.set(cacheLargeArtifacts);
     }
-
-    public String finalName() {
-        return getFinalName().get();
-    }
+    
+     /**
+     * @param fileCollection The file collection from which to retrieve the last file.
+     * @return The last file in the file collection, or null if the file collection is empty.
+     */
+    public static File getLastFile(FileCollection fileCollection) {
```

Comment:
I read your answer to Max but I still don't understand why this change is needed? Why do you need this method?